cloudferrylib/os/storage/cinder_storage.py

```diff
@@ -60,6 +61,16 @@ def get_client(self, params=None):
             insecure=self.config.cloud.insecure
         )
 
+    def get_endpoint(self):
+        return self.identity_client.get_endpoint_by_service_type(
+            service_type='volume',
+            endpoint_type='publicURL')
+
+    def get_endpoint_host(self):
+        m = re.search("//([^:^/]*)", self.get_endpoint())
+        # so cute regex
+        return m.group(1)
```

Comment:
What if there's no port specified in endpoint URL, e.g. endpoint is 'http://cinder-endpoint.mycloud.com/'?..